test/export/test_converter.py

```diff
@@ -966,6 +1010,20 @@ def func2(*args):
         # Cannot script variable length inputs.
         self._check_equal_ts_ep_converter(func2, tuple(values), ["trace"])
 
+    def test_convert_script_object(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.tq = _empty_tensor_queue()
+
+            def forward(self, x: torch.Tensor):
+                self.tq.push(x)
+                torch.ops._TorchScriptTesting.queue_push(self.tq, x.cos())
+                return torch.ops._TorchScriptTesting.queue_pop(self.tq), self.tq.pop()
+
+        inp = (torch.randn(2, 3),)
+        self._check_equal_ts_ep_converter(M(), inp, ["script"])
```

Comment:
does jit.trace not support?

Reply:
Right, both push call does not return anything, so trace does not work.